app/views/users/index.html.erb

```diff
@@ -31,4 +31,4 @@
 <%= link_to 'New User', new_user_path %>
 <%= link_to 'New Meal', new_meal_path %>
 <%= link_to 'Show All Meals', meals_path %>
-<%= link_to 'Pick What To Eat NOW!', meals_path%>
+<%= link_to 'Pick What To Eat NOW!', @meal_path %>
```

Comment:
Was this a typo? There doesn't appear to be an instance variable called `@meal_path` defined in users#index.